Sources/SwiftProtobuf/Google_Protobuf_Any+Registry.swift

```diff
@@ -45,7 +45,7 @@ internal func typeName(fromURL s: String) -> String {
     }
   }
 
-  return s[typeStart..<s.endIndex]
+  return String(s[typeStart..<s.endIndex])
```

Comment:
On Swift 4, converting a StringSlice to a String does seem to be a real operation.  But I don't think there's any "cost" to worry about.  I think that it's the same total work as with Swift 3, Swift 4 just exposes it as two separate operations (marking the slice and copying it into a String).  I don't think we want to return a StringSlice instead, so we don't really have a choice anyway.